src/scss/components/navigation/_table-of-contents.scss

```diff
@@ -0,0 +1,85 @@
+.table-of-contents {
+  @include components-container-max-width();
+  margin-left: 0;
+  margin-right: 0;
+
+  @include breakpoint($breakpoint-m) {
+    @include component-side-padding();
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .has-sidebar & {
+    @include component-negative-side-margin;
+  }
+
+  // In rare cases TPR Service page can be rendered without sidebar.
+  // In these cases remove the left and right paddings from table of contents.
+  .service-header__container:not(.has-sidebar) & {
+    @include breakpoint($breakpoint-m) {
+      padding-left: 0;
+      padding-right: 0;
+    }
+  }
+}
+
+.table-of-contents__container {
+  @include component-margin-top();
+  background-color: $hdbt-color-white;
+  border-left: 8px solid var(--hdbt-color-gold--primary);
+  max-width: $content-width-max;
+  padding: $spacing-double $spacing-and-half;
+}
+
+.table-of-contents__title {
+  @include font-size(24px, 28px);
+  margin-bottom: 0;
+  margin-top: 0;
+
+  @include breakpoint($breakpoint-l) {
+    @include font-size(36px, 42px);
+  }
+}
+
+.table-of-contents__list {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.table-of-contents__item {
+  padding-top: $spacing;
```

Comment:
I think we should exclude this padding-top from the first element to make handling table of contents without title easier. Instead of the first element having this padding-top, the space between title and table-of-contents__list should be set when there's title entered.
![image](https://user-images.githubusercontent.com/1191667/141274428-6df1b93d-db43-47b5-8e31-9443dc7e4252.png)